mmv1/products/workstations/WorkstationCluster.yaml

```diff
@@ -155,6 +155,12 @@ properties:
           The service attachment is created when private endpoint is enabled.
           To access workstations in the cluster, configure access to the managed service using (Private Service Connect)[https://cloud.google.com/vpc/docs/configure-private-service-connect-services].
         output: true
+      - !ruby/object:Api::Type::Array
+        name: 'allowedProjects'
+        item_type: Api::Type::String
+        description: |
```

Comment:
If there is a default, can we add `default_from_api: true` to allow the provider to pick up the API's default?

Reply:
As FYI since I was looking at implementing this earlier, this field's default is empty, since it's a list of _additional_ project IDs, not the full list.

Reply:
I've added the `default_from_api: true` to the `allowedProjects` field as the description on the API property says: By default, the workstation cluster's project and the VPC host project (if different) are allowed.